scripts/db_entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+#
+
+command="$1"
+timeout=10
+hostname=localhost
```

Comment:
Any way to skip this when running the migration in Lambda? The leads to an unnecessary 10 second delay when the Lambda starts.

Reply:
The commit message where this was added has some context: f936dad9

First, we do not actually wait 10 seconds every run. We probe the port, and if it's open, we move on immediately. Ten seconds is lost only when the connection fails completely, and then go storage tests will fail on the step. (On my machine using gnu netcat, `nc -z localhost 5432` averaged  2ms over 10,000 tries).

I was getting builds that would intermittently succeed and then fail when I re-ran them, or vice-versa, due to the migration container failing to connect to the database in the test job. I believe the cause is a race condition between the migration container and the postgres container in the test executor: if the migration container stands up faster than postgres (it usually does) then flyway exits with error and the storage tests fail.

I added this timeout routine and... so far so good? So it's actually there explicitly for the migration container. A clean task doesn't need it at all, since the RDS instance in DEV is accepting connections 24/7. 

If we could define dependencies in the test executor the way we do with docker-compose, that would be ideal, but I wasn't able to find a way to do that. I'll create a research ticket to look into it again.

Reply:
I understand the need for the script when testing the containers in CI.

And to correct my earlier comment, I meant when running the db migration in ECS, not Lambda. Maybe I'm misunderstanding, but since the entrypoint script is baked into the image, the ECS task will still try to connect to localhost:5432 when it starts up. Since the ECS task does not connect to RDS using localhost, those connection attempts will fail until it reaches the 10-second timeout. The 10-second delay can be observed in the ECS task's logs.

Reply:
Ah yes, true. I'll modify this to take arguments.

Reply:
Flyway docs describe an env var [FLYWAY_CONNECT_RETRIES](https://flywaydb.org/documentation/envvars#FLYWAY_CONNECT_RETRIES) that sounds like it will handle this more elegantly. I'm testing this now.